### PR TITLE
Fixed double parsing of Candy-generated messages as XHTML

### PR DESCRIPTION
**Pull request by daniellim2000**
parseAndCropXHTML() currently parses already parsed XHTML messages and this causes an error.

### CODE REVIEW

**Comment by daniellim2000** [comment]
Hope this can resolve candy-chat/candy/#446

File: src/util.js [hunk 1]
```diff
@@ -110,9 +110,12 @@ Candy.Util = (function(self, $){
...
 	 */
 	self.parseAndCropXhtml = function(str, len) {
-		return $('<div/>').append(self.createHtml($(str).get(0), len)).html();
+		var msg = str;
+		if ( typeof str !== 'string' ) {
+			msg = self.createHtml($(str).get(0), len);
+		}
+		return $("<div/>").append(msg).html();
 	};
-
```
Comment (benlangfeld) [inline src/util.js]: Mistakenly removed line.

**Comment by benlangfeld** [comment]
Ooops, merged this to dev. Cherry-picked to master at https://github.com/candy-chat/candy/commit/53b0cbbe27409650483674b98250ce5b9a793a11

**Comment by daniellim2000** [comment]
sorry uh i was afk for the past few days

**Pull request merged**